ocpmodels/trainers/forces_trainer.py

```diff
@@ -294,6 +294,13 @@ def train(self):
                 if i % self.config["cmd"]["print_every"] == 0:
                     print(self.meter)
 
+                iters_per_epoch = self.config["optim"].get(
+                    "iters_per_epoch", -1
+                )
+                if iters_per_epoch > 0:
+                    if i > iters_per_epoch:
+                        break
+
```

Comment:
Let's maybe rename `iters_per_epoch` to something like `eval_every`.

We probably don't want to cut each training epoch shorter in order to evaluate more frequently. The model should see the entire dataset in each epoch.

`i` counts iterations within each epoch. We can have another variable `iters` that keeps track of global no. of iterations, and then do something like:

```python
if iters % eval_every == 0:
    if self.val_loader is not None:
        val_metrics = self.validate(split="val", epoch=epoch)
```

Reply:
Makes sense. Borrowed from how @weihua916 has it implemented, but this is a good point.